src/services/completions.ts

```diff
@@ -1050,6 +1051,12 @@ namespace ts.Completions {
                 }
             }
 
+            if (isMetaProperty(node) && (node.keywordToken === SyntaxKind.NewKeyword || node.keywordToken === SyntaxKind.ImportKeyword)) {

```

Comment:
can you merge from master one more time. i think these should already be in master now.